specification/communication/data-plane/Microsoft.CommunicationServicesChat/preview/2020-11-01-preview3/communicationserviceschat.json

```diff
@@ -0,0 +1,1589 @@
+        },
+        "x-ms-examples": {
+          "Send read receipt": {
+            "$ref": "./examples/Conversations_SendChatReadReceipt.json"
+          }
+        }
+      }
+    },
+    "/chat/threads/{chatThreadId}/messages": {
+      "post": {
+        "tags": [
+          "Messages"
+        ],
+        "summary": "Sends a message to a thread.",
+        "operationId": "SendChatMessage",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "chatThreadId",
+            "description": "The thread id to send the message to.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "sendChatMessageRequest",
+            "description": "Details of the message to send.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SendChatMessageRequest"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Message sent, the `Location` header contains the URL for the newly sent message.",
+            "schema": {
+              "$ref": "#/definitions/SendChatMessageResult"
+            }
+          },
+          "401": {
+            "description": "Unauthorized",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "x-ms-error-response": true
+          },
+          "403": {
+            "description": "Forbidden",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "x-ms-error-response": true
+          },
+          "429": {
+            "description": "Too many requests",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "x-ms-error-response": true
+          },
+          "503": {
+            "description": "Service unavailable",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-examples": {
+          "Send Message": {
+            "$ref": "./examples/Messages_SendChatMessage.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Messages"
+        ],
+        "summary": "Gets a list of messages from a thread.",
+        "operationId": "ListChatMessages",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "chatThreadId",
+            "description": "The thread id of the message.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "maxPageSize",
```

Comment:
guidelines name it differently, `$maxpagesize`

Reply:
updated